Returning a javax.swing.ImageIcon as a code cell results generates an…

kpaharelau opened the pull request:
… exception. #4052

I have used idea from http://stackoverflow.com/questions/12533543/converting-a-imageicon-array-to-a-bufferedimage

kpaharelau committed:
Returning a javax.swing.ImageIcon as a code cell results generates an exception. #4052

I have used idea from http://stackoverflow.com/questions/12533543/converting-a-imageicon-array-to-a-bufferedimage

scottdraves commented on plugin/jvm/src/main/java/com/twosigma/beaker/jvm/serialization/ImageIconSerializer.java:
```diff
@@ -15,6 +15,7 @@
  */
 package com.twosigma.beaker.jvm.serialization;
 
+import java.awt.*;
```
import \* violates style.

kpaharelau committed:
Returning a javax.swing.ImageIcon as a code cell results generates an exception. #4052

I>import * violates style.
FIXED

kpaharelau committed:
Returning a javax.swing.ImageIcon as a code cell results generates an exception. #4052

>scottdraves commented 9 days ago
>also, it causes the java icon to pop up.
>you fixed this problem for R a long time ago, can you fix it for Java too?
>
>otherwise it works.

DONE
